extensions/amp-iframe/amp-iframe-vs-native-iframe.md

```diff
@@ -0,0 +1,22 @@
+# amp-iframe vs native iframe
+
+## Background
+
+amp-iframe has been an AMP component since ~2015. It was designed to make native iframes more secure while
+enforcing restrictions like its position outside the top 75% or 600px in an AMP document to not degrade user experience. With the migration of amp-iframe to Bento, we want to assess if we retire amp-iframe component in favor of native iframes, which now support new features like lazy loading.
+
+## So - can we retire amp-iframe?
+
+No, we still need the amp-iframe component. There are several features that cannot be implemented using native iframes, like the ones discussed below.
+
+### AMP Cache
+
+Content is preloaded and cached by the AMP cache and displayed in the AMP viewer when an AMP document is requested. With native iframes, we cannot ensure that the content is not preloaded or cached.
```

Comment:
@kristoferbaxter not sure if I've captured this accurately?

Reply:
Can you elaborate on "we cannot ensure that the content is not preloaded or cached."

Are there strategies for making this work with lazy loaded iframes that could be broadly adopted? What are the downsides of these approaches?